ambari-web/app/controllers/wizard/step7/assign_master_controller.js

```diff
@@ -65,7 +65,8 @@ App.AssignMasterOnStep7Controller = Em.Controller.extend(App.BlueprintMixin, App
     
     switch (action) {
       case 'ADD':
-        if (hostComponent.componentName === "HIVE_SERVER_INTERACTIVE") {
+        this.clearRecommendations();
+        if (hostComponent.componentName == "HIVE_SERVER_INTERACTIVE") {
```

Comment:
Why not "===" ?

Reply:
"===" should be used

Reply:
This was intended change happened while cherry-picking https://github.com/apache/ambari/pull/286/commits/e0e187824e0aa5f60031a2c4e8a83764d04066cf commit from 2.6 to trunk. I will revert it back

Reply:
Done